sycl/test/basic_tests/event.cpp

```diff
@@ -1,7 +1,10 @@
-// REQUIRES: opencl
+// REQUIRES: opencl || level0
```

Comment:
https://github.com/intel/llvm/commit/215f591ade97aed616c4094a467e5dcbebc45d6b changed this feature name.

level0 -> level_zero

Please, fix ASAP.

Reply:
up for review https://github.com/intel/llvm/pull/2235